ReportUtilityBills/measure.rb

```diff
@@ -150,6 +200,11 @@ def energyPlusOutputRequests(runner, user_arguments)
       end
     end
 
+    if args[:include_monthly_bills]
+      # force a eplusout_monthly.msgpack from which we get timestamps
+      result << OpenStudio::IdfObject.load('Output:Variable,*,Site Outdoor Air Drybulb Temperature,monthly;').get
+    end
```

Comment:
Why do you need to do this? The measure is already requesting monthly fuel outputs.

Reply:
Oh, I see. The monthly fuel requests are output meter requests and this is an output variable request. You should be able to remove this output variable request and obtain the timestamps in the eplusout.msgpack file, which is where output meter results end up. (Don't ask me why it's like this.)

Reply:
Isn't it possible that eplusout.msgpack doesn't have monthly timestamps though? I.e., when a building consumes only electricity and has detailed electric rates defined.